esp_hosted_ng/host/Makefile

```diff
@@ -8,17 +8,7 @@ KERNEL := /lib/modules/$(shell uname -r)/build
 # Architecture
 ARCH := arm
 
-#Default interface is sdio
-MODULE_NAME=esp32_sdio
-
-#Targets passed overrrides default value
-ifeq ($(target), sdio)
-	MODULE_NAME=esp32_sdio
-endif
-
-ifeq ($(target), spi)
-	MODULE_NAME=esp32_spi
-endif
+MODULE_NAME=esp32
```

Comment:
We always want users to use either sdio or spi. This helps them to clear their head that they have used only a specific device tree. Also, we omit sdio symbols while building spi and vice versa. So this Makefile change cannot be taken.

Reply:
> We always want users to use either sdio or spi. This helps them to clear their head that they have used only a specific device tree. 
I can write the documentation to back this up as it makes more sense to do it like this.
> Also, we omit sdio symbols while building spi and vice versa. So changes in Makefile related to this, cannot be taken.
In this commit, I have already renamed the functions so that it is bisectable.

Reply:
It's not about is it correct or not. it is whether we want to load extra code to kernel, which is dead code. For example, while using SDIO, SPI code is dead code loaded, which is not a correct way to implement.